apollo-client/src/main/java/com/ctrip/framework/apollo/internals/DefaultConfig.java

```diff
@@ -45,7 +45,8 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class DefaultConfig extends AbstractConfig implements RepositoryChangeListener {
-  private static final Logger logger = LoggerFactory.getLogger(DefaultConfig.class);
+
+  private static final Logger logger = DeferredLoggerFactory.getLogger(DefaultConfig.class);
```

Comment:
Please also change LocalFileConfigRepository and RemoteConfigRepository to use `DeferredLogger`, I tested in my local environment and found the actual cause won't be printed in eager load mode